README.md

```diff
@@ -1 +1,62 @@
-# WEB-SUBMIT-2022-ACCRA
\ No newline at end of file
+# Web Summit 2022 Accra
+
+Live preview [here](https://jerryowusu.github.io/Web-Summit-Accra/). 
+
+## Screenshots
+![Example screenshot](/src/images/desktop-screenshot.png)
+
+## Contents
+* [Homepage](#homepage)
+* [Program](#program)
+* [Partners](#partners)
+* [Contact](#contact)
+* [Mobile menu](#mobile)
+* [Click more for expand speakers](#speakers)
+
+## General info
+This project is my first Capstone.
+	
+## Technologies
+Project is created with:
+* HTML
+* CSS
+* Bootstrap
+* Javascript
+
+	
+## Setup
+To run this project, install it locally using:
+$ cd Desktop
+$ git clone https://github.com/jerryowusu/Web-Summit-Accra.git
+
+## Author
+
+👤 **Jerry Owusu**
+
+- Github : [@jerryowusu](https://github.com/jerryowusu)
+- Twitter: [@jerry owusu](https://twitter.com/jerryowusu1)
+- LinkedIn : [@Jeremiah Owusu](https://www.linkedin.com/in/jeremiah-owusu-b50a70173/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests  are always welcome!
+
+Feel free to check the [issues page]https://github.com/jerryowusu/Web-Summit-Accra/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- A special thanks to Cindy Shin author of the original design.
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] The license link is broken. Kindly fix it.

Reply:
Resolved. Thanks